specs/trip_dispatch_spec.rb

```diff
@@ -90,3 +90,18 @@
     end
   end
 end
+
+
+describe "request_trip method" do
+  it "returns a trip" do
+    dispatcher = RideShare::TripDispatcher.new
```

Comment:
There are many more things you should be testing for the `request_trip` method:

- Does the trip reference the correct passenger and driver?
- Was the trip added to the trip lists of the driver and passenger?
- What if there is no available driver?

The last is an important edge case, and is exactly the sort of situation where software could do something unexpected. You might set up the test as follows:

```ruby
it 'raises an error if no driver is available' do
  dispatcher = RideShare::TripDispatcher.new
  dispatcher.drivers.each do |driver|
    # This may require an attr_writer in Driver
    driver.status = :UNAVAILABLE
  end
  proc {
    dispatcher.request_trip(2)
  }.must_raise StandardError
end
```